docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -50,6 +47,15 @@ async function renderApiDocumentation() {
 		newlineKind: "lf",
 		uriRoot: "/docs/apis",
 		includeTopLevelDocumentHeading: false, // This will be added automatically by Hugo
+		packageFilterPolicy: (apiPackage) => {
```

Comment:
This is the primary change. Filter out packages with the scope `@fluid-internal` from those processed when generating Markdown docs.